source/include/jobs.h

```diff
@@ -249,7 +368,38 @@ JobsStatus_t Jobs_MatchTopic( char * topic,
  * the buffer up to one less than the buffer size.  The topic is
  * ended with a NUL character.
  *
- * @note The thingName parameter does not need a NULL terminator.
+ * @note The thingName parameter does not need a NUL terminator.
```

Comment:
NULL-terminated string actually sounds like a misnomer considering that the proper label for the terminating character is NUL.

Reply:
Interestingly, it is @dan4thewin who made me aware of the correct spellings for the `\0` character 😛

Reply:
haha same here